### ISSUE TITLE
TextSecureAttachment.java Exception "No length specified!"

### ISSUE THREAD

**Issue by cholin** [WhisperSystems/Signal-Android#5331]
I have:
- [x] searched open and closed issues for duplicates: maybe related to #3882 (closed / unresolved)
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports
- [ ] checked if maybe cyanogenmod 13.0 issue
### Bug description

Every time after I start signal (after successfully entering my password) it loads my messages and immediately crashes. My android signal is linked to a signal chromium app (Chromium: 49.0.2623.87-1, Signal Private Messenger: 0.1.6) . Only android signal seems affected.
### Steps to reproduce

Hard to tell. Occurred only once (5 minutes ago):
1. send images in group chats via android signal
2. wait until it crashes for the first time
3. signal will now crash everytime after starting (after password input) with the following exception:

```
03-12 19:19:56.633  3542  3558 E AndroidRuntime: FATAL EXCEPTION: JobConsumer-1
03-12 19:19:56.633  3542  3558 E AndroidRuntime: Process: org.thoughtcrime.securesms, PID: 3542
03-12 19:19:56.633  3542  3558 E AndroidRuntime: Theme: themes:{}
03-12 19:19:56.633  3542  3558 E AndroidRuntime: java.lang.IllegalArgumentException: No length specified!
03-12 19:19:56.633  3542  3558 E AndroidRuntime:    at org.whispersystems.textsecure.api.messages.TextSecureAttachment$Builder.build(TextSecureAttachment.java:80)
03-12 19:19:56.633  3542  3558 E AndroidRuntime:    at org.thoughtcrime.securesms.jobs.PushSendJob.getAttachmentsFor(PushSendJob.java:78)
03-12 19:19:56.633  3542  3558 E AndroidRuntime:    at org.thoughtcrime.securesms.jobs.PushGroupSendJob.deliver(PushGroupSendJob.java:132)
03-12 19:19:56.633  3542  3558 E AndroidRuntime:    at org.thoughtcrime.securesms.jobs.PushGroupSendJob.onSend(PushGroupSendJob.java:82)
03-12 19:19:56.633  3542  3558 E AndroidRuntime:    at org.thoughtcrime.securesms.jobs.SendJob.onRun(SendJob.java:43)
03-12 19:19:56.633  3542  3558 E AndroidRuntime:    at org.thoughtcrime.securesms.jobs.MasterSecretJob.onRun(MasterSecretJob.java:18)
03-12 19:19:56.633  3542  3558 E AndroidRuntime:    at org.whispersystems.jobqueue.JobConsumer.runJob(JobConsumer.java:76)
03-12 19:19:56.633  3542  3558 E AndroidRuntime:    at org.whispersystems.jobqueue.JobConsumer.run(JobConsumer.java:46)
```

Workaround:
1. activate airplane mode
2. search for image in group chat that is still loading/sending
3. delete this image
4. deactivate airplane mode
5. from now everything is fine
### Screenshots

none
### Device info

**Model:**  motorola Moto E (cm_condor)

**Android version:** : 6.0.1 (11d82ef7a5, cm_condor-userdebug 6.0.1 MMB29V 11d82ef7a5 test-keys)

**Signal version:**     : Signal 3.13.1
### Link to debug log

**Android signal:** https://gist.github.com/anonymous/88d3e5094ceb2db300da#file-cat-log-L27
**Chromium app:** https://gist.github.com/cholin/79f91c6c77452257fd5e
### Suggestions

Maybe catch exception and just don't try to render failed attachment messages?

**Comment by 2-4601** [comment]
> #3882 (closed / unresolved)

That has actually been resolved: 973ed7686bb1d6f6344611d48778cbfb2d3ba388

**Comment by cholin** [comment]
Sorry, seems I missed 973ed76. Furthermore I just noticed my chromium app version was outdated (v0.1.6). I updated to v0.3.2 but can reproduce the issue because no image got stucked in my group chats.

**Comment by Trolldemorted** [comment]
This issue is fixed by PR #5926.
